frontend/fingertips-frontend/components/organisms/LineChartTable/index.tsx

```diff
@@ -1,19 +1,77 @@
 'use client';
 
 import { Table } from 'govuk-react';
-import { sortHealthDataByDate } from '@/lib/chartHelpers/formatChartValues';
 import { HealthDataForArea } from '@/generated-sources/ft-api-client';
+import styled from 'styled-components';
 
 interface TableProps {
-  data: HealthDataForArea[];
+  data: HealthDataForArea;
   headings: string[];
 }
 
+interface LineChartTableData {
+  period: number;
+  count: number;
+  value: number;
+  lower: number;
+  upper: number;
+}
+
+const StyledAlignCenterHeader = styled(Table.CellHeader)({
+  textAlign: 'center',
+  padding: '0.3125em',
+});
+
+const StyledAlignRightHeader = styled(Table.CellHeader)({
+  textAlign: 'right',
+  paddingTop: '0',
+});
+
+const StyledTable = styled(Table)({
+  marginBottom: '0.9375em',
+});
+
+const StyledTableCell = styled(Table.Cell)({
+  textAlign: 'left',
```

Comment:
Looking at the designs the numerical columns are right aligned. Only the period seems to be left aligned

Reply:
Nice that's looking lot more like the designs.